config/locales/en.yml

```diff
@@ -39,12 +39,12 @@ en:
     retention_4: Retention 4
     completion: Completion
   withdrawal_reasons:
-    left_teaching_profession: "Left teaching profession"
-    moved_school: "Moved school"
-    mentor_no_longer_mentor: "Mentor no longer being mentor"
-    school_left_fip: "School left FIP"
-    career_break: "Career break"
-    passed_induction: "Passed Induction"
+    left_teaching_profession: Left teaching profession
+    moved_school: Moved school
+    mentor_no_longer_mentor: Mentor no longer being mentor
+    school_left_fip: School left FIP
+    career_break: Career break
+    passed_induction: Passed Induction
```

Comment:
What's the difference? If it is significant, then we have more things to clean up in this file...